data-dashboard/server/trpc/index.ts

```diff
@@ -7,18 +7,15 @@ const t = initTRPC.context<TContext>().create();
 const isAuthed = t.middleware(({ ctx, next }) => {
   try {
     const { req } = ctx;
-    const token = req.headers.authorization?.split(' ')[1]; // actual token
-
-    if (!token) throw new TRPCError({ code: 'UNAUTHORIZED' });
-
-    verify(token, 'chairesesunindividuomuyguapo');
+    const token = String(req.cookies['access-token']);
```

Comment:
The Authorization header does not persist and needs to be updated every request. Maybe in the future I can add a function that retrieves the token from the context, but I thought it would be useful to take advantage of cookie persistance.